challenge_serving_drinks/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>challenge_serving_drinks</name>
+  <version>0.0.0</version>
+  <description>The challenge_serving_drinks package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <maintainer email="jannolunenburg@gmail.com">Janno Lunenburg</maintainer>
```

Comment:
@SamAlexandrov you are the maintainer...

Reply:
Done :)